.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "lib/rpc_spec"]
```

Comment:
This comment is a reminder to undo this change once the rpc_spec includes the new feature in develop

Reply:
Updated rpc_spec commit hash, and restored the .gitmodules accordingly.